packages/pluginutils/test/dataToEsm.ts

```diff
@@ -2,6 +2,13 @@ import test from 'ava';
 
 import { dataToEsm } from '../';
 
+test('support bigint and symbol', (t) => {
+  t.is(
+    dataToEsm({ bigint: 0, symbol: Symbol.for('') }),
```

Comment:
I'd like to see a few different values (within the same `t` test func) tried here for sanity check